index.html

```diff
@@ -0,0 +1,841 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/styles/layouts.css">
+    <link rel="stylesheet" href="/styles/style.css">
+    <link rel="shortcut icon" href="/icons/twitter.png" type="image/x-icon">
+    <title>Twitter Challenge</title>
+</head>
+<body>
+    <div class="center-wrapper three-column-grid__expand-two">
+        <aside class="left-menu asides-padding">
+            <nav>
+                <ul>
+                    <img id="logo" class="margin-bottom-md" src="/icons/twitter.png" alt="Twitter icon.">
+                    <li>
+                        <a class="two-column-grid_expand-two gap-md align-center" href="#">
+                            <svg fill="currentColor" height="26px" width="26px" alt="Home icon.">
+                                <g>
```

Comment:
whats the meaning of the g tag?

Reply:
The `<g>` is a grouping tag, its meant to group its children. So, multiple "drawings" can be grouped to make one picture/icon.